packages/redux/localstorage/hooks/useSettings.ts

```diff
@@ -41,7 +41,7 @@ export const useSettings: UseSettings = (context) => {
   )
 
   const updateSlippageTolerance = useCallback(
-    (slippageTolerance: number) => {
+    (slippageTolerance: string) => {
```

Comment:
Any reason for this to be a string? This will break a bunch of other stuff which expects numer. Percent is a small range so can safely be represented as a number always.